addons/website_event/views/event_type_views.xml

```diff
@@ -7,14 +7,16 @@
         <field name="inherit_id" ref="event.view_event_type_form"/>
         <field name="arch" type="xml">
             <xpath expr="//div[hasclass('oe_title')]" position="after">
-                    <span name="website_menu">
-                        <label for="website_menu" string="Website Submenu"/>
+                <div class="grid o_inner_group col-lg-4 d-flex flex-wrap w-auto">
```

Comment:
Ok to make this one a div to match what we do for the event form.

However, same remark about the additional divs for the checkboxes.

Also, would add `name="event_menu_configuration"` here as well, to ease potential xpathing.